graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/OutputTest.java

```diff
@@ -402,7 +480,26 @@ void typesFromMultipleSchemas() {
     void subtype() {
         assertGeneratedContentContains(
                 "subtype",
-                ".getId(),DSL.row(_a_customer.FIRST_NAME,",
-                ").mapping(Functions.nullOnAllNull(CustomerName::new))).mapping(Functions.nullOnAllNull(Customer::new))");
+                """
+                        public class QueryDBQueries {
+                            public static Customer queryForQuery(DSLContext _iv_ctx, SelectionSet _iv_select) {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                return _iv_ctx
+                                        .select(queryForQuery_customer())
+                                        .from(_a_customer)
+                                        .fetchOne(_iv_it -> _iv_it.into(Customer.class));
+                            }
+                            private static SelectField<Customer> queryForQuery_customer() {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                return DSL.row(
+                                        _a_customer.getId(),
+                                        DSL.row(
+                                                _a_customer.FIRST_NAME,
+                                                _a_customer.LAST_NAME
+                                        ).mapping(Functions.nullOnAllNull(CustomerName::new))
+                                ).mapping(Functions.nullOnAllNull(Customer::new));
+                            }
+                        }
+                        """);
```

Comment:
Usikker på om denne testen ikke bare dupliserer en av de wrapper-testene over, men:

```suggestion
                "ctx.select(queryForQuery_customer())",
                "return DSL.row(_a_customer.getId(),_a_customer.getId(),DSL.row(_a_customer.FIRST_NAME,"
```

Reply:
den er veldig lik innerNestedRow. 

```
type Query {
  query: Customer
}

type Customer @table {
  wrapper: Wrapper
}

type Wrapper {
  id: ID
}
```

vs. 

```
type Query {
    query: Customer
}

type Customer @table {
  id: ID!
    name: CustomerName
}

type CustomerName {
  firstName: String @field(name: "first_name")
  lastName: String @field(name: "last_name")
}
```

Er det noe med at det er "vanlige" felter istedenfor ID som gjør at det er verdt en egen test?
Jeg heller mot å fjerne denne

Reply:
For struktureringen har ikke typen noe å si, fordi typen blir avgjort når man allerede vet hvor det skal. Det er altså en separat og uavhengig logikk som setter opp den biten, dermed tror jeg at å fjerne testen er greit.